### DISCUSSION TITLE
Creating a new string

### DISCUSSION THREAD

**Discussion by Congealed** [vfsfitvnm/frida-il2cpp-bridge#533] [Q&A] [answered]
Sorry for the basic question, but I cant find out how to Instantiate a simple string, Lets say i have a function
`this.method("WriteAsync").invoke("1", 1);`
but the "1" strings pops an incorrect parameter type (The type is System.String).

any help appreciated

**Comment by thinhbuzz** [comment, marked as answer]
Il2Cpp.string('string')

**Reply by Congealed** [reply]
Thanks!
